nsv13/code/controllers/subsystem/overmap_mode.dm

```diff
@@ -361,6 +367,51 @@ SUBSYSTEM_DEF(overmap_mode)
 	if(mode.difficulty <= 0)
 		mode.difficulty = 1
 
+/datum/controller/subsystem/overmap_mode/proc/force_mode(new_mode)
+	if(!mode || !mode_initialised)
+		mode = new_mode
+		forced_mode = new_mode
+		return initial(mode.name)
```

Comment:
This will also often runtime because mode is not expected to be a path at any time after `ssovermap_mode.Initialize()`, while `mode_initialised` is not set to true until the game starts (so, will runtime during the entire time period between those two events)